plugins/env/db/migrate/20150527210505_add_environment_variables.rb

```diff
@@ -0,0 +1,22 @@
+class AddEnvironmentVariables < ActiveRecord::Migration
+  def change
+    create_table :environment_variables do |t|
+      t.string :name, null: false
+      t.string :value, null: false
+      t.string :environment_variable_group_id, null: false
+      t.integer :deploy_group_id
+    end
+    add_index :environment_variables, [:environment_variable_group_id, :name, :deploy_group_id], unique: true, name: "environment_variables_unique_deploy_group_id"
+
+    create_table :environment_variable_groups do |t|
+      t.string :name, null: false
+    end
+    add_index :environment_variable_groups, :name, unique: true
+
+    create_table :stage_environment_variable_groups do |t|
+      t.integer :stage_id, :environment_variable_group_id, null: false
```

Comment:
I'm using a unique index on both for the first one ... but yeah same reason: I want to see the actual structure and not the railsy abstraction